src/qt/glcanvas.cpp

```diff
@@ -440,26 +494,33 @@ bool wxGLCanvas::SwapBuffers()
     return true;
 }
 
+bool wxGLCanvas::QtCanPaintWithoutActivePainter() const
+{
+    return true;
+}
+
 /* static */
-bool wxGLCanvas::ConvertWXAttrsToQtGL(const int *wxattrs, QGLFormat &format)
+bool wxGLCanvas::ConvertWXAttrsToQtGL(const wxGLAttributes &wxGLAttrs, const wxGLContextAttrs wxCtxAttrs, QGLFormat &format)
 {
-    if (!wxattrs)
-        return true;
+    const int *glattrs = wxGLAttrs.GetGLAttrs();
+    const int *ctxattrs = wxCtxAttrs.GetGLAttrs();
 
     // set default parameters to false
     format.setDoubleBuffer(false);
     format.setDepth(false);
     format.setAlpha(false);
-    format.setStencil(false);
+    format.setStencil(true);
```

Comment:
This is inconsistent with the comment above now. As [the docs](https://doc.qt.io/qt-5/qglformat.html#setStencil) say that it's enabled by default, maybe we should just remove this line?

```suggestion
```

Reply:
This should be set to false, now corrected.